ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -958,6 +958,9 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
+    /// The UIView of the `accessoryView`.
+    @objc open var accessoryButtonView: UIView? { return accessoryTypeView }
```

Comment:
Yea I was stuck on naming too, I chose it based off of the delegate method. I can change it to that, since I couldn't use `accessoryTypeView` nor `accessoryView`